documentation/docs/30-advanced/20-hooks.md

```diff
@@ -126,7 +126,7 @@ export async function handleFetch({ request, fetch }) {
 }
 ```
 
-**Credentials**
+#### Credentials
```

Comment:
Let's revert it for now, a h4 is rendered super tiny on svelte.dev (maybe we should adjust that over there; not sure)
```suggestion
**Credentials**
```